docs/utilities.md

```diff
@@ -863,22 +863,22 @@ localhost,{KDBBASEPORT}+5,rdb,rdb3,${TORQAPPHOME}/appconfig/passwords/accesslist
 localhost,{KDBBASEPORT}+6,rdb,rdb4,${TORQAPPHOME}/appconfig/passwords/accesslist.txt,1,1,180,,${KDBCODE}/processes/rdb.q,1,,q
 ```
 
-##### $KDBAPPCONFIG/settings/rdb.q
+**$KDBAPPCONFIG/settings/rdb.q**
 
 **Ensure** **`.rdb.subfiltered: 0b`**
 
 ---
 
-#### 3) Data striping for general subscribers
+**3) Data striping for general subscribers**
 
-##### {subscriber}.q
+**{subscriber}.q**
 
 Add the conditions required (using the data hash striping function).
 
 ```q
 ...
 // Subscribe to all tables but with custom conditions
-conditions:1!flip`tabname`filters`columns!(`trade`quote;("sym in .ds.stripe[sym;0]";"sym in .ds.stripe[sym;1]");("";""))
+conditions:1!flip`tabname`filters`columns!(`trade`quote;(".ds.stripe[sym;0]";".ds.stripe[sym;1]");("";""))
```

Comment:
Is this definitely a correct example? I would've thought the common case was for a subscriber to subscribe to all tables for the same segment, rather than trade=segment 0, quote=segment 1

Reply:
Yes this is a correct example. I was trying to show that it is possible to have custom conditions where a single general subscriber has the flexibility to subscribe to different tables in different segments.